recipes/biomechanical-analysis-framework/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "biomechanical-analysis-framework" %}
+{% set cxx_name = "libbiomechanical-analysis-framework" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ami-iit/biomechanical-analysis-framework/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: fb12cae8da4146c1bb39f7f71e5e95b43c8e8714a1065fa18dbb307c1e8f4d15
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ cxx_name }}
```

```suggestion
  # {{ cxx_name }}
  - name: libbiomechanical-analysis-framework
```

See https://github.com/conda-forge/gz-sim-yarp-plugins-feedstock/pull/3 and the links there, we can't use the jinja macro in the name of the packages.